www/js/main.js

```diff
@@ -433,13 +434,19 @@ function setBound() {
     };
     switch (getUa()) {
         case "Android":
+            bound.height = window.innerHeight;
+            bound.width = window.innerWidth;
+            //bound.zoom = screen.availWidth / bound.width;
```

Comment:
@daydreamheart why you comment `bound.zoom` ? what is it anyway?

Reply:
It was originally with iOS, but I tested on Android too. 
It made the bricks a little bit bigger, but since I changed the size of the bound there is no change when that line is added, that's the reason I commented it.